content/vault/BlogPost1.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2023-07-02
+title: Welcome to unspent.space
+tags:
+  - welcome
+  - newbie
+  - open source
+  - franchise
+
+---
+
+# Welcome!
+
+
```

Comment:
I like what you've done here, but maybe you could customise the blog to make it unique ?